docs/pages/versions/v45.0.0/sdk/filesystem.md

```diff
@@ -259,13 +259,13 @@ Get metadata information about a file, directory or external content/asset.
 
 #### Arguments
 
-- **fileUri (_string_)** -- URI to the file or directory. It may be e.g. URI returned by [`CameraRoll.getPhotos()`](https://reactnative.dev/docs/cameraroll.html#getphotos). See [supported URI schemes](#supported-uri-schemes-1).
+- **fileUri (_string_)** -- URI to the file or directory. See [supported URI schemes](#supported-uri-schemes-1).
 
 - **options (_object_)** -- A map of options:
 
   - **md5 (_boolean_)** -- Whether to return the MD5 hash of the file. `false` by default.
 
-  - **size (_boolean_)** -- Whether to include the size of the file if operating on a source from [`CameraRoll.getPhotos()`](https://reactnative.dev/docs/cameraroll.html#getphotos) (skipping this can prevent downloading the file if it's stored in iCloud, for example). The size is always returned for `file://` locations.
+  - **size (_boolean_)** -- Explicitly specify that the file size should be included - skipping this can prevent downloading the file if it's stored in iCloud, for example. The size is always returned for `file://` locations.
```

Comment:
```suggestion
  - **size (_boolean_)** -- Explicitly specify that the file size should be included. For example, skipping this can prevent downloading the file if it's stored in iCloud The size is always returned for `file://` locations.
```

The re-phrasing matches the [Docs writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md).